woocommerce/woocommerce-android
# Issue/6217 flux update to store searched products

Fixes #6217 by updating the FluxC hash to [this PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2352) which stores product search results in the database.

To test:

* Clear storage then log in to the app
* Create an order
* Tap to add a product
* Search for a product that isn't in the first page of products
* Add it to the order
* Ensure it still appears when you return to the main order creation screen
